docs/settings/security-settings.asciidoc

```diff
@@ -239,3 +239,107 @@ The format is a string of `<count>[ms\|s\|m\|h\|d\|w\|M\|Y]` (e.g. '20m', '24h',
 ============
 
 |===
+
+[float]
+[[audit-logging-settings]]
+===== Audit logging settings
+
+You can enable audit logging to support compliance, accountability and security. When enabled Kibana will capture:
+
+- Who performed an action
+- What action was performed
+- When it occurred
+
+For more details and a reference of audit events see <<xpack-security-audit-logging>>.
+
+[cols="2*<"]
+|===
+| `xpack.security.audit.enabled`
+| Set to `true` to enable audit logging for security events. *Default:* `false`
+
+| `xpack.security.audit.ignore_filters[]` (beta)
```

Comment:
These settings now are a bit of a blend between the legacy and new loggers. What do you think about introducing just the `xpack.security.audit.enabled` setting first, but then below get into the configuration for the new beta logger?

This would also let us add a heading to encapsulate all of the beta settings, where we can add a single `beta[]` indicator to warn users that everything in the section is a beta feature (instead of having `beta()` everywhere)

Reply:
That's a good idea, will do